### ISSUE TITLE
Feature for copying math equation in unicode

### ISSUE THREAD

**Issue by Bibeksharma5** [nvaccess/nvda#20124]
### Is your feature request related to a problem? Please describe.

Hello! I'm using NVDA 2026.1. My one request for NVDA math reading feature. We can copy a math equation by pressing Ctrl+C when we focus on any equation. NVDA provides the provision for copying the equation in math ML, LaTeX and some others. One option should be added there. Which is unicode. I always type the equation in equation editor in word in unicode format. If I encounter any equation on the web or even in MS word, I can simply copy and paste without changing the mode for typing equation. Regards.

### Describe the solution you'd like

There should be provision to copy the equation in unicode format. For EG: if there is an equation for finding root of a quadratic equation, the copied math equation (in unicode) should be like:
(-b+-\sqrt(b^2-4ac))/2a

**Comment by seanbudd** [comment]
Please request this from the mathcat maintainers, we can't fix this in NVDA directly

https://github.com/daisy/MathCAT

**Comment by Bibeksharma5** [comment]
OK. I'll do it.
